setup.py

```diff
@@ -9,7 +9,7 @@
         # client:
         "requests",
         "urllib3<2.0.0, >=1.26.0",
-        "pydantic<2.0.0",
+        "pydantic>=2.0.0",
```

Comment:
It is good to provide a generic version so we don't have a strict requirement here. But can you confirm that you have tested everything on 2.0.0?

Reply:
HI Li, all tests pass on 2.0.0 
![Screenshot 2024-05-15 at 8 56 09 AM](https://github.com/marqo-ai/py-marqo/assets/16517734/ab71e7a4-4cf9-4f22-8ce2-8244f311b752)
![Screenshot 2024-05-15 at 8 56 27 AM](https://github.com/marqo-ai/py-marqo/assets/16517734/3fd810b4-c2ee-4e1e-8036-37e905da61a0)